### PR TITLE
update: hide create and add new button if only one product creation allowed

### PR DESCRIPTION
**Pull request by shohag121**
This PR is related to https://github.com/weDevsOfficial/dokan-pro/issues/1334

### CODE REVIEW

File: templates/products/new-product.php [hunk 1]
```diff
@@ -265,7 +265,17 @@
 
                             <div class="dokan-form-group dokan-right">
                                 <?php wp_nonce_field( 'dokan_add_new_product', 'dokan_add_new_product_nonce' ); ?>
+                                <?php
+                                $display_create_and_add_new_button = true;
+                                if ( method_exists( '\DokanPro\Modules\Subscription\Helper', 'get_vendor_remaining_products' )  ) {
```
Comment (nurul-umbhiya) [inline templates/products/new-product.php:270]: don't check with method exists, follow below code to check if a module is active or not
```
function_exists( 'dokan_pro' ) && dokan_pro()->module->is_active( 'product_subscription' )
```

File: templates/products/new-product.php [hunk 2]
```diff
@@ -265,7 +265,17 @@
...
+                                    if ( 2 > \DokanPro\Modules\Subscription\Helper::get_vendor_remaining_products( dokan_get_current_user_id() ) ) {
```
Comment (nurul-umbhiya) [inline templates/products/new-product.php:271]: bhai don't user condition like this, use condition like below
```
\DokanPro\Modules\Subscription\Helper::get_vendor_remaining_products( dokan_get_current_user_id() ) <= 1
or
\DokanPro\Modules\Subscription\Helper::get_vendor_remaining_products( dokan_get_current_user_id() ) < 2

File: templates/products/new-product.php [hunk 3]
```diff
@@ -265,7 +265,17 @@
...
+                                        $display_create_and_add_new_button = false;
```
Comment (nurul-umbhiya) [inline templates/products/new-product.php:272]: what if \DokanPro\Modules\Subscription\Helper::get_vendor_remaining_products( dokan_get_current_user_id() ) < 1 ? do we suppose to display  any buttons ??

File: templates/products/tmpl-add-product-popup.php [hunk 4]
```diff
@@ -153,7 +153,17 @@
...
+                $display_create_and_add_new_button = true;
+                if ( method_exists( '\DokanPro\Modules\Subscription\Helper', 'get_vendor_remaining_products' )  ) {
+                    if ( 2 > \DokanPro\Modules\Subscription\Helper::get_vendor_remaining_products( dokan_get_current_user_id() ) ) {
+                        $display_create_and_add_new_button = false;
+                    }
+                }
+                if ( $display_create_and_add_new_button ) :
+                ?>
                 <input type="submit" id="dokan-create-and-add-new-product-btn" class="dokan-btn dokan-btn-theme" data-btn_id="create_and_new" value="<?php esc_attr_e( 'Create & add new', 'dokan-lite' ) ?>">
+                <?php endif; ?>
```
Comment (nurul-umbhiya) [inline templates/products/tmpl-add-product-popup.php:166]: where is the Create New button if count is equal to 1 ?

File: templates/products/new-product.php (hunk 1 above)
Comment (shohag121) [reply templates/products/new-product.php]: Changed chacking as you instructed.

File: templates/products/new-product.php (hunk 3 above)
Comment (shohag121) [reply templates/products/new-product.php:272]: We hide the entire product form.

File: templates/products/tmpl-add-product-popup.php (hunk 4 above)
Comment (shohag121) [reply templates/products/tmpl-add-product-popup.php:166]: Above the changed code, on line 155 Bhai.

File: templates/products/new-product.php (hunk 2 above)
Comment (shohag121) [reply templates/products/new-product.php]: Changed the condition Bhai.

**Review by nurul-umbhiya — APPROVED** [review]

**Pull request merged**